babel.config.js

```diff
@@ -21,7 +22,16 @@ module.exports = function (api) {
       [
         "module:react-native-dotenv",
         {
+          envName: "APP_ENV",
+          moduleName: "@env",
+          path: ".env",
+          blocklist: null,
+          allowlist: null,
+          blacklist: null, // DEPRECATED
+          whitelist: null, // DEPRECATED
```

Comment:
If its deprecated do we still need to include it?

Reply:
In the example they use this, so I was copying the example. The thing is flaky either way so I was hoping that by providing all the options it would be less flaky.